roles/monitoring/tasks/cluster-logging.yaml

```diff
@@ -140,6 +140,7 @@
     kind: Namespace
     name: "{{ V4M_LOGGING_NAMESPACE }}"
     wait: true
+    wait_timeout: 600
```

Comment:
Is this something we can create a variable for or have already? Also is this in seconds? Needs to be clear if we're exposing this to the customer. If not this is fine.

Reply:
We don't have a variable for this already but wondered that also and it is seconds. 
Three other namespace delete task timeouts just like these already existed in other sections of the code before the one update and two new ones I added in this PR for monitoring and logging. I searched the current code and docs and see no mention of these timeouts so would conclude that we don't expose it.

Reply:
That's what I needed to know. Looks good then.